README.markdown

```diff
@@ -1013,6 +1013,8 @@ Directives
 * [init_worker_by_lua](#init_worker_by_lua)
```

Comment:
BTW, this file is automatically generated (you can check out the comment at the beginning of this file).

You should edit `doc/HttpLuaModule.wiki` file instead and use the `ngx-releng` tool chain to update this markdown file automatically from the `.wiki` file.